plugins/kotlin/j2k/k1.new.post-processing/src/org/jetbrains/kotlin/idea/j2k/post/processing/allProcessings.kt

```diff
@@ -131,15 +131,13 @@ private val inspectionLikePostProcessingGroup = InspectionLikeProcessingGroup(
 private val cleaningUpDiagnosticBasedPostProcessingGroup = DiagnosticBasedPostProcessingGroup(
     removeUselessCastDiagnosticBasedProcessing,
     removeUnnecessaryNotNullAssertionDiagnosticBasedProcessing,
-    fixValToVarDiagnosticBasedProcessing
 )
 
 private val inferringTypesPostProcessingGroup = NamedPostProcessingGroup(
     KotlinNJ2KServicesBundle.message("processing.step.inferring.types"),
     listOf(
         InspectionLikeProcessingGroup(
             processings = listOf(
-                VarToValProcessing(),
```

Comment:
Please don't forget that `VarToValProcessing` is also used in K2. You should remove it from K2 and fix the failing tests. Currently tests are failing in K2 and not in K1 because in K1 we have an additional `LocalVarToValInspectionBasedProcessing` that is now doing more work.